floor_generator/lib/misc/extension/type_converters_extension.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:analyzer/dart/element/type.dart';
+import 'package:dartx/dartx.dart';
+import 'package:floor_generator/misc/annotations.dart';
+import 'package:floor_generator/value_object/type_converter.dart';
+import 'package:source_gen/source_gen.dart';
+
+extension TypeConvertersExtension on Iterable<TypeConverter> {
+  @nullable
+  TypeConverter get closestOrNull {
+    return sortedByDescending((typeConverter) => typeConverter.scope.index)
+        .firstOrNull;
+  }
+
+  @nullable
+  TypeConverter getClosestOrNull(DartType dartType) {
+    return sortedByDescending((typeConverter) => typeConverter.scope.index)
+        .firstOrNullWhere(
+            (typeConverter) => typeConverter.fieldType == dartType);
```

Comment:
❓Does `is` work here?
```suggestion
            (typeConverter) => typeConverter.fieldType is dartType);
```